### PR TITLE
Custom "exceeds goal" behaviour for AU frontline campaign

### PR DESCRIPTION
**Pull request by joelochlann**
## below goal
Picture 468

## over goal
Picture 469

## further over goal
Picture 470

### CODE REVIEW

**Review by ionamckendrick — APPROVED** [review]
Looking good; thank you for the extra design changes!

**Review by jranks123 — APPROVED** [review]

**Comment by jranks123** [comment]
I do worry that this isn't that clear. For me, it isn't obvious enough that the black line is the target amount. But I'm just here to approve the code :)

**Comment by joelochlann** [comment]
Happy to iterate on this design tomorrow, so @ionamckendrick, @jranks123, @jlieb10 feel free to discuss what your ideal design would look like, without time constraints.

Just getting this one out quick so we have something in place for when we exceed the goal later today/tonight

**Comment by ionamckendrick** [comment]
@joelochlann @jranks123 I also have other ideas for the design; this was purely to reuse something we already built for the US end of year campaign, so we could get it out quickly. And it's better than the alternative where it could look broken. I will provide alternatives to chat about tomorrow!

**Comment by joelochlann** [comment]
Yep. When we have that discussion, let's delimit all the possible variations, design & implement them, and then forbid alternatives.

e.g.
1. ticker simply stops once goal reached
2. ticker stops once goal reached, and displays thank you message 
3. ticker continues after goal, with black line shifting further to the left with more contribs,
.... etc

File: static/src/javascripts/projects/common/modules/commercial/ticker.js [hunk 1]
```diff
@@ -93,17 +88,10 @@ const populateGoal = (parentElement: HTMLElement) => {
...
     const parentElement = document.querySelector(parentElementSelector);
 
-    const tickerElementSelector =
-        total > goal ? '.js-ticker-over-goal' : '.js-ticker-under-goal';
+    const tickerElementSelector = '.js-ticker-amounts';
 
     if (parentElement && parentElement instanceof HTMLElement) {
-        const heightClass =
-            total > goal ? 'epic-ticker--short' : 'epic-ticker--tall';
-        parentElement.classList.add(heightClass);
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/commercial/ticker.js]: I think this `heightClass` bit might have been the only change you needed to make in this file, other than setting `allowToExceed` to `true`

**Pull request merged**